ui/js/page/filePage/view.jsx

```diff
@@ -116,7 +116,7 @@ class FilePage extends React.PureComponent {
                   : uriIndicator}
               </div>
               <div className="card__actions">
-                <FileActions uri={uri} />
+                <FileActions uri={uri} claim={claim} />
```

Comment:
Are you sure an actual second resolve is triggered? I.e. is an API call actually made or are you just seeing the action dispatch?

Either way, the claim shouldn't be passed. If we are doing an unnecessary extra resolve, that should be fixed elsewhere.